src/device/prims_ll.h

```diff
@@ -855,8 +856,8 @@ class Primitives<T, RedOp, Fan, Direct, ProtoLL, P2p, isNetOffload>:
       int eltInLine = EltPerLine < nelem ? EltPerLine : nelem;
 
       DataLoader dl;
-      ncclLLFifoLine line[MaxRecv];
-      uint64_t data, peerData;
+      // ncclLLFifoLine line[MaxRecv];//unused variable - compiler warning
+      uint64_t data;                  //unused variable - compiler warning
```

Comment:
Might consider this so we see what was removed.
```suggestion
      uint64_t data/*, peerData*/;    //unused variable - compiler warning
```